File: facerecapi/tim.js
// // Input timestamp in the format "2023-09-02T05:12:28.015Z"
// const timestamp = "2023-09-02T05:12:28.015Z";

// // Create a JavaScript Date object from the timestamp
// const date = new Date(timestamp);

// // // Format the date and time in a more human-readable format
// // const options = {
// //   year: "numeric",
// //   month: "2-digit",
// //   day: "2-digit",
// //   hour: "2-digit",
// //   minute: "2-digit",
// //   second: "2-digit",
// //   fractionalSecondDigits: 3,
// //   timeZoneName: "short",
// // };

// const formattedDate = date.toLocaleDateString();
// const time=date.toLocaleTimeString()
// console.log(time);

// console.log(formattedDate);
const time= new Date()
console.log(time.toLocaleDateString());

File: facerecapi/tim.py
from datetime import datetime,date
# timestamp = datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%S.%fZ')
# print(timestamp)
# t=datetime.now()
# print(t.strftime("%H:%M:%S"))
t=str(date.today().day)+"/"+str(date.today().month)+"/"+str(date.today().year)[2:]

print(t)